index.html

```diff
@@ -179,6 +324,145 @@ <h2 class="about-me-header">Software Developer</h2>
           <p class="work-paragraph">Work</p>
         </div>
       </div>
+      <section class="my-recent-work">
```

Comment:
this section or maybe another parent `div` should have the display `grid` that houses all works section. while each works container should then use a display `flex` to organize its children elements